client/app/utils/__tests__/index.test.js

```diff
@@ -0,0 +1,35 @@
+import { randomString, setCsrfToken, getPusher } from '../index';
+import { JSDOM } from 'jsdom';
+
+
+describe('randomString, setCsrfToken, getPusher tests ', () => {
+  it('should return random string', () => {
+    expect(typeof randomString()).toBe('string');
+  });
+
+
+  it('should be equal to pusher key and cluster when getPusher is called', () => {
```

Comment:
I think you'll also want to test the `return null` case as well, when `window.pusher` does into exist.